src/openfl/text/TextField.hx

```diff
@@ -1414,27 +1415,11 @@ class TextField extends InteractiveObject implements IShaderDrawable {
 		
 		if (__textEngine.antiAliasType == ADVANCED && __textEngine.gridFitType == PIXEL) {
 			
-			var smoothingEnabled = untyped (renderSession.context).imageSmoothingEnabled;
```

Comment:
i think that never worked anyways, because we're supposed to cehck all these 3-4 properties instead of just one, so i simply made it work the same way as in the other places - without checks. every canvas render call (re)sets the smoothing property anyway